src/libraries/System.DirectoryServices.Protocols/src/System/DirectoryServices/Protocols/common/SearchResults.cs

```diff
@@ -31,24 +30,7 @@ public Uri[] Reference
             }
         }
 
-        public DirectoryControl[] Controls
-        {
-            get
-            {
-                if (_resultControls == null)
-                {
-                    return Array.Empty<DirectoryControl>();
-                }
-
-                DirectoryControl[] controls = new DirectoryControl[_resultControls.Length];
-                for (int i = 0; i < _resultControls.Length; i++)
-                {
-                    controls[i] = new DirectoryControl(_resultControls[i].Type, _resultControls[i].GetValue(), _resultControls[i].IsCritical, _resultControls[i].ServerSide);
-                }
-                DirectoryControl.TransformControls(controls);
-                return controls;
-            }
-        }
+        public DirectoryControl[] Controls => Array.Empty<DirectoryControl>();
```

Comment:
Looks like in here we were previously handing back a copy so that the actual results object wouldn't be mutated but that seems like it would change with this right? I know property doesn't have a setter but one could get the directory controls back and then change them. Also, Seems like now we are losing the TransformControls call.

Reply:
This shouldn't change any behavior: _resultControls is always null. Did I misread?

Reply:
You are right.